movement/sample_data.py

```diff
@@ -0,0 +1,116 @@
+"""Module for fetching and loading sample datasets.
+
+This module provides functions for fetching and loading sample data used in
+tests, examples, and tutorials. The data are stored in a remote repository
+on GIN and are downloaded to the user's local machine the first time they
+are used.
+"""
+
+from pathlib import Path
+
+import pooch
+import xarray
+import yaml
+
+from movement.io import load_poses
+
+# URL to the remote data repository on GIN
+# noinspection PyInterpreter
+DATA_URL = (
+    "https://gin.g-node.org/neuroinformatics/movement-test-data/raw/master"
+)
+
+# Save data in ¬/.movement/data
+DATA_DIR = Path("~", ".movement", "data").expanduser()
+# Create the folder if it doesn't exist
+DATA_DIR.mkdir(parents=True, exist_ok=True)
+
+# Fetch newest sample metadata
+Path.unlink(DATA_DIR / "poses_files_metadata.yaml", missing_ok=True)
```

Comment:
Also, here you first delete the existing file, and then you download the new one.
I'm going to be paranoid and say what if the internet connection is down? Then your pooch request will fail and you will end up with no metadata files whatsoever. I think the prudent thing to do would be the following:
- rename the old metadata file to sth like "temp_poses_file_metadata.yaml", for safekeeping
- try to download the newest file, and if that fails rename the temp file back to its original filename, and raise a warning (this can be done with a try/except clause)

Reply:
if the above logic get's too complex, you can encapsulate it in a function (sth like `fetch_metadata`), and add a line like `metadata = fetch_metadata(file_name="poses_files_metadata.yaml")`, which will also solve your DRY problem.